HousingRepairsOnlineApi/Startup.cs

```diff
@@ -79,6 +79,13 @@ public void ConfigureServices(IServiceCollection services)
                 return new SendAppointmentConfirmationEmailUseCase(notifyGateway, emailConfirmationTemplateId);
             });
 
+            services.AddTransient<IAppointmentConfirmationSender, AppointmentConfirmationSender>(s =>
+            {
+                var sendAppointmentConfirmationEmailUseCase = s.GetService<ISendAppointmentConfirmationEmailUseCase>();

```

Comment:
Each of the `ISendAppointmentConfirmationXXXUseCase`s are already registered (`ISendAppointmentConfirmationEmailUseCase` on line 76 and `ISendAppointmentConfirmationSmsUseCase` on line 70).
This would allow those types to be resolved automatically when required and this lambda instantiating `AppointmentConfirmationSender` is not required - just the registration alone will suffice.